scripts/completion/dvc.zsh

```diff
@@ -234,28 +240,34 @@ _arguments \
   '*::args:->args'
 
 case $words[1] in
-  init)      _arguments $_dvc_global_options   $_dvc_init      ;;
-  destroy)   _arguments $_dvc_global_options   $_dvc_destroy   ;;
   add)       _arguments $_dvc_global_options   $_dvc_add       ;;
-  import)    _arguments $_dvc_global_options   $_dvc_import    ;;
+  # cache?
   checkout)  _arguments $_dvc_global_options   $_dvc_checkout  ;;
-  run)       _arguments $_dvc_global_options   $_dvc_run       ;;
-  pull)      _arguments $_dvc_global_options   $_dvc_pull      ;;
-  push)      _arguments $_dvc_global_options   $_dvc_push      ;;
+  commit)    _arguments $_dvc_global_options   $_dvc_commit    ;;
+  # daemon?
+  config)    _arguments $_dvc_global_options   $_dvc_config    ;;
+  destroy)   _arguments $_dvc_global_options   $_dvc_destroy   ;;
+  # diff?
   fetch)     _arguments $_dvc_global_options   $_dvc_fetch     ;;
-  status)    _arguments $_dvc_global_options   $_dvc_status    ;;
-  repro)     _arguments $_dvc_global_options   $_dvc_repro     ;;
-  remove)    _arguments $_dvc_global_options   $_dvc_remove    ;;
-  move)      _arguments $_dvc_global_options   $_dvc_move      ;;
   gc)        _arguments $_dvc_global_options   $_dvc_gc        ;;
-  config)    _arguments $_dvc_global_options   $_dvc_config    ;;
-  remote)    _arguments $_dvc_global_options   $_dvc_remote    ;;
-  metrics)   _arguments $_dvc_global_options   $_dvc_metrics   ;;
+  import)    _arguments $_dvc_global_options   $_dvc_import    ;;
+  init)      _arguments $_dvc_global_options   $_dvc_init      ;;
   install)   _arguments $_dvc_global_options   $_dvc_install   ;;
-  root)      _arguments $_dvc_global_options   $_dvc_root      ;;
   lock)      _arguments $_dvc_global_options   $_dvc_lock      ;;
-  unlock)    _arguments $_dvc_global_options   $_dvc_unlock    ;;
+  metrics)   _arguments $_dvc_global_options   $_dvc_metrics   ;;
+  move)      _arguments $_dvc_global_options   $_dvc_move      ;;
   pipeline)  _arguments $_dvc_global_options   $_dvc_pipeline  ;;
-  commit)    _arguments $_dvc_global_options   $_dvc_commit    ;;
+    # pkg?
+  pull)      _arguments $_dvc_global_options   $_dvc_pull      ;;
+  push)      _arguments $_dvc_global_options   $_dvc_push      ;;
+  remote)    _arguments $_dvc_global_options   $_dvc_remote    ;;
+  remove)    _arguments $_dvc_global_options   $_dvc_remove    ;;
+  repro)     _arguments $_dvc_global_options   $_dvc_repro     ;;
+  root)      _arguments $_dvc_global_options   $_dvc_root      ;;
+  run)       _arguments $_dvc_global_options   $_dvc_run       ;;
+  # tag?
+  status)    _arguments $_dvc_global_options   $_dvc_status    ;;
+  unlock)    _arguments $_dvc_global_options   $_dvc_unlock    ;;
   unprotect) _arguments $_dvc_global_options   $_dvc_unprotect ;;
+  # version?
```

Comment:
I believe it is not taking any arguments for now and I don't think it will in the nearest future, so we could safely skip it.
```suggestion
```